### PR TITLE
Bump electron dependencies

### PR DESCRIPTION
**Pull request by djsmith85**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [X] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

To stay up-to-date with electron release schedule, bumping electron dependencies

Depends on: https://github.com/bitwarden/jslib/pull/579

Asana task: https://app.asana.com/0/1200804338582616/1201506018477651/f

## Code changes
* **jslib:** Pull in changes done in https://github.com/bitwarden/jslib/pull/579
* **src/package.json:** 
   * Update `electron-log` to `4.4.1
   * Update `electron-store` to `8.0.1`
   * Update `electron-updater` to `4.6.1` 
* **src/package-lock.json:** Changes after running `npm i` in src/
* **package.json:** 
   * Update `electron-rebuild` to `3.2.5` 
   * Update `electron-builder` to `22.14.5` 
* **package-lock.json:** changes after running `npm i` in /

## Testing requirements
Requires full regression

## Before you submit
- [X] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Review by Hinton — COMMENTED** [review]
Build is failing but code looks good.

File: tslint.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "object-literal-sort-keys": false,
     "object-literal-shorthand": [true, "never"],
     "prefer-for-of": false,
+    "quotemark": [true, "single"],
```
Comment (differsthecat) [inline tslint.json:41]: I think this was removed with the prettier changes

Comment (differsthecat) [reply tslint.json:41]: https://github.com/bitwarden/directory-connector/pull/194/files#diff-9db81b3f77968b3ad243c25f280c4e80dd20b2e5770bf5606ccdb4ecb8af9ce8L41

**Comment by djsmith85** [comment]
Closing this in favor of https://github.com/bitwarden/directory-connector/pull/197, something was messed up when merging prettier. I assume because some of the config changes happened on the Apply prettier commit, which we ignored using `git merge -Xours 096196fcd512944d1c3d9c007647a1319b032639`

**Pull request closed without merging**